server/data/statementsClient.ts

```diff
@@ -20,10 +22,13 @@ export default class StatementsClient {
             inner join report r on s.report_id = r.id   
           where s.user_id = $1
           and s.deleted is null
-          order by status, r.incident_date desc`,
-      values: [userId],
+          order by status, r.incident_date desc
+          offset $2
+          limit $3`,
+      values: [userId, offset, limit],
     })
-    return result.rows
+
+    return buildPageResponse(result.rows, page)
```

Comment:
`buildPageResponse` is a generic function which will create an object with pagination metadata and results (trimming off unnecessary `totalCount` from each row)